tests/hpc/rasdaemon.pm

```diff
@@ -47,8 +47,13 @@ sub run {
 
     assert_script_run('! ras-mc-ctl --status');
 
+    # Try to start rasdaemon. May need a restart on aarch64.
     systemctl('start rasdaemon');
-    systemctl('is-active rasdaemon');
+    if (systemctl('is-active rasdaemon', ignore_failure => 1)) {
+        systemctl('restart rasdaemon');
```

Comment:
I think would be better to do `is-active rasdaemon` few times before restart, as mentioned last time. Restart after first failed check could mask potential issue.  Could you please try `script_retry` before restart?

Reply:
I tried that for 10 seconds before and it always failed.

Reply:
hmmm but indeed the solution you propose could hide some real issues, so the approach you propose would require active review of passing tests (to avoid false positives; I guess that's proper qa term?).

Reply:
As @ge0r has proposed, a softfail might be better. As you write in the description :
> the test for rasdaemon in virtualized env. has very limitted purpose and
>  is meant only as a sanity check for the isolated package.
So I don't know if we could mark this as a bug unless we also run it on real HW, or somehow check if this is a qemu bug.

Reply:
Also `script_retry` will `die` on timeout, so we can't recover from that. From my testing with a 10s timeout, `script_retry` will always die if `is-active` fails the first time.